internal/store/store.go

```diff
@@ -42,27 +43,27 @@ func NewBlockStore(db dbm.DB) *BlockStore {
 }
 
 // Base returns the first known contiguous block height, or 0 for empty block stores.
-func (bs *BlockStore) Base() int64 {
+func (bs *BlockStore) Base() (int64, error) {
 	iter, err := bs.db.Iterator(
 		blockMetaKey(1),
 		blockMetaKey(1<<63-1),
 	)
 	if err != nil {
-		panic(err)
+		return 0, pkg_errors.Wrap(err, "Error while iterating blockMetaKeys")
```

Comment:
Is there any reason for using `pkg_errors.Wrap` over `fmt.Errorf`?

Reply:
we should definitely prefer fmt.Errorf, as we don't really use pkg_errors.Cause.